mkosi.images/base/mkosi.conf.d/10-arch/mkosi.conf

```diff
@@ -12,6 +12,7 @@ InitrdInclude=initrd/
 Environment=
         PKG_URL="https://gitlab.archlinux.org/daandemeyer/systemd"
         PKG_BRANCH="strip"
+        PKG_COMMIT="733045c8f6b0acf2f0cfaac207500483a678f4d2"
```

Comment:
The problem is that those commits will get out of of date immediately. How would we even keep them up to date?

Maybe a time-based solution would be better? E.g. take the timestamp of the commit of systemd that is being built, and do the equivalent of `git log --pretty=oneline <branch> --since <timestamp> --reverse|head -n1` and use that to select the packaging? (I.e. the first packaging commit later than the commit to sources.)

Reply:
@keszybz I'll write a github actions workflow to automatically create PRs to update the commits. I really think we should be explicit here and encode specific commit hashes.

Reply:
Switched to git submodules instead + dependabot to do the updates for us